src/main/webapp/script.js

```diff
@@ -100,21 +100,84 @@ window.addEventListener('load', () => {
 
   getOptions();
 
-  function initCalendar() {
-    new tui.Calendar('#calendar', {
-      defaultView: 'week',
-      useCreationPopup: true,
-      useDetailPopup: true,
-      disableDblClick: true,
-      disableClick: true,
-      isReadOnly: true,
-      scheduleView: ['time'],
-      taskView: false,
-      week: {
-        workweek: true,
-      },
+  if (document.head.id == 'scheduler') {
+    function initCalendar() {
+      new tui.Calendar('#calendar', {
+        defaultView: 'week',
+        useCreationPopup: true,
+        useDetailPopup: true,
+        disableDblClick: true,
+        disableClick: true,
+        isReadOnly: true,
+        scheduleView: ['time'],
+        taskView: false,
+        week: {
+          workweek: true,
+        },
+      });
+    }
+
+    initCalendar();
+  } else {
+    document.getElementById('submit-plan').addEventListener('submit', () => {
+      /**
+       * Gets results from /planner servlet to results
+       */
+      async function getPlan() {
+        const courseContainer = document.getElementById('order-area');
+        createSpinner(courseContainer);
+        const response = await fetch(
+            `/planner?selectedClasses=${selected.toString()}&semesters=${
+                document.getElementById('semesters').value}`);
+        courseList = await response.json();
+        createTable(courseList, courseContainer);
+      }
+
+      /**
+       * Creates spinner to signify loading
+       * @param {Object} courseContainer container for course list
+       */
+      function createSpinner(courseContainer) {
+        const spinner = document.createElement('i');
+        spinner.setAttribute('class', 'fas fa-sync fa-spin');
+        courseContainer.innerText = '';
+        courseContainer.appendChild(spinner);
+      }
+
+      /**
+       * Creates table from a 2D array
+       * @param {Object} tableData 2D array with separation of courses
+       * @param {Object} courseContainer container for course list
+       */
+      function createTable(tableData, courseContainer) {
+        courseContainer.innerText = '';
+        const table = document.createElement('table');
+        const tableBody = document.createElement('tbody');
+        table.setAttribute('class', 'table table-hover mb-0;');
+        let i = 1;
+        tableData.forEach(function(rowData) {
```

Comment:
1. please use arrow functions for consistency
2. You can have an automatic counter

```
tableData.forEach((rowData, i) => {
  // note that i starts at 0, but you can just add 1 to it when you need.
});